### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by sneakyweasel**
Various cleanup / bug fixes

### CODE REVIEW

File: src/components/AppPhoton.vue [hunk 1]
```diff
@@ -106,7 +107,7 @@ export default class AppPhoton extends Vue {
     return {
       opacity: `${this.displayOpacity ? this.opacity : 1}`,
       'transform-origin': `${this.width / 2}px ${this.height / 2}px`,
-      transform: `rotate(${this.particle.direction}deg)`
+      transform: `rotate(${this.displayDirection ? this.particle.direction : 0}deg)`
```
Comment (stared) [inline src/components/AppPhoton.vue]: It is a bad pattern. Create a getter or a utility function, don't put undefined-logics in string templates.

File: src/components/GamePage/GameToolbox.vue [hunk 2]
```diff
@@ -41,7 +41,7 @@ export default class GameToolbox extends Vue {
   }
 
   computedClass(cell: Cell): string {
-    return this.isAvailable(cell) ? 'inactive' : 'active';
+    return this.isAvailable(cell) ? 'active' : 'inactive';
```
Comment (stared) [inline src/components/GamePage/GameToolbox.vue]: Do you know the pattern: `:style="{active: 'expression'}"`?
https://vuejs.org/v2/guide/class-and-style.html

In most cases it is cleaner

Comment (sneakyweasel) [reply src/components/GamePage/GameToolbox.vue]: This case is a bit more complex

Comment (stared) [reply src/components/GamePage/GameToolbox.vue]: Exclusive classes are not the best pattern. (E.g. should be only active, if not 'active' then non active. Or the other way, whichever is simpler.)

If really needed `:style="{active: 'isAvailable(cell)', inactive: '!isAvailable(cell)'}"`.

**Pull request merged**